README.md

```diff
@@ -0,0 +1,546 @@
+Drive or in similar cloud storage.
+* If a password is published by mistake, *you need to change the
+password* (with everything that this entails). It is *not* enough to
+remove/reverse the commit or submit a new commit with the password
+removed.
+* Code should use placeholders that point to:
+    * Local read-protected files, possibly located outside of the
+    Git repository file structure to avoid accidental inclusion as
+    part of the repository,
+    * Environment variables,
+    * or some sort of secured (possibly remote) storage.
+* The documentation (README/INSTALL, whichever is most appropriate)
+should mention how to instantiate those variables/files, etc.
+
+### Testing
+
+(TODO: Write me)
+
+## How we use GitHub
+
+In order to maximize exposure, and to facilitate collaborations with
+users and other organizations, we have opted to use GitHub and the
+infrastructure that GitHub provides for publishing all our publically
+accessible software. This also means that we will be using Git (rather
+than Subversion, Mercurial, CVS or any other code revision system) for
+keeping track of source code and documents relating to software that we
+make available on GitHub.
+
+GitHub provides excellent support for doing code reviews (more on that
+below), collecting issues (bug reports) connected with a project, and
+for organising work tasks ("projects" in GitHub speak).
+
+On GitHub, we have an Educational Account called "NBISweden"
+("NBIS" was taken) which acts like an umbrella for all our various
+repositories. Code repositories are public (available to the world),
+but we have the ability to create private repositories that are only
+available to members of NBISweden, *but only if there are very special
+circumstances that require this*. The ELIXIR Open Source Principles say
+“Start a project in the open from the very first day, in a publicly
+accessible, version controlled repository [...] The longer a project is
+run in a closed manner, the harder it is to open source it later”.
+
+The source code that we produce are Public Records, and as such should
+be made publically available as Open Source. This is a requirement
+within ELIXIR and for projects funded by the Swedish government.
+
+To contribute to NBISweden repositories, or to create repositories
+there, you will need to set up a GitHub account for yourself and let the
+admins of NBISweden know. Current admins include *Mikael Borg*, *Jonas
+Hagberg*, and *Johan Viklund*.
+
+## How we use Git
+
+When appropriate, we use the *Git-Flow branching model*. This is a way
+of using Git branches as a help in the development cycle.
+
+For more in-depth descriptions of Git-Flow, see
+
+* [http://nvie.com/posts/a-successful-git-branching-model/](http://nvie.com/posts/a-successful-git-branching-model/)
+* [http://danielkummer.github.io/git-flow-cheatsheet/](http://danielkummer.github.io/git-flow-cheatsheet/)
+
+With Git-Flow, branches are categorised into
+
```

Comment:
I divided the big list with its text into a tiny list plus free-standing paragraphs. Looks better, and the tiny list gives a sort of an overview.